delivery_ups_oca/models/ups_request.py

```diff
@@ -32,6 +32,7 @@ def __init__(self, carrier):
     def _process_reply(
         self, url, data=None, method="post", query_parameters=None,
     ):
+        data = data if data else {}
```

Comment:
Why this can be `None`?

```suggestion
        data = data or {}
```

Reply:
When trying to cancel a shipment without set this parameter https://github.com/OCA/delivery-carrier/blob/13.0/delivery_ups_oca/models/ups_request.py#L257 i have seen this problem.